<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountSecurity</class>
 <widget class="QWidget" name="AccountSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="account_recovery">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="recover_user_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="recover_user_email">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recover_user_btn">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
         </property>
         <property name="text">
          <string>Recover Username</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="recover_pass_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="recover_pass_email">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recover_pass_username">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #545454;
color:white;
border-style:none;</string>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recover_pass_btn">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}</string>
         </property>
         <property name="text">
          <string>Recover Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
